tests/typing/bad/mult_errs1.catala_en

```diff
@@ -0,0 +1,52 @@
+> Include: common.catala_en
+
+```catala
+scope S:
+  definition z equals [
+    Structure { -- i: 3.4 -- e: Int content x };
+    Structure { -- i: 4.1 -- e: y };
+    Structure { -- i: 5 -- e: Dat content |1970-01-01| }
+  ]
+  definition a equals number of (z ++ z) / 2
+```
+
+```catala-test-inline
+$ catala Typecheck
+┌─[ERROR]─
+│
+│  Error during typechecking, incompatible types:
+│  ─➤ decimal
+│  ─➤ integer
+│
+│ This expression has type decimal:
+├─➤ tests/typing/bad/mult_errs1.catala_en:6.23-6.26:
+│   │
+│ 6 │     Structure { -- i: 3.4 -- e: Int content x };
+│   │                       ‾‾‾
+│
+│ Expected type integer coming from expression:
+├─➤ tests/typing/bad/common.catala_en:8.18-8.25:
+│   │
+│ 8 │   data i content integer
+│   │                  ‾‾‾‾‾‾‾
+└─
+┌─[ERROR]─
+│
+│  Error during typechecking, incompatible types:
+│  ─➤ decimal
+│  ─➤ integer
+│
+│ This expression has type decimal:
+├─➤ tests/typing/bad/mult_errs1.catala_en:7.23-7.26:
+│   │
+│ 7 │     Structure { -- i: 4.1 -- e: y };
+│   │                       ‾‾‾
+│
+│ Expected type integer coming from expression:
+├─➤ tests/typing/bad/common.catala_en:8.18-8.25:
+│   │
+│ 8 │   data i content integer
+│   │                  ‾‾‾‾‾‾‾
+└─
```

Comment:
In this case, it's pretty relevant though 🤔

Reply:
Same remark as above :)